front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -148,34 +149,86 @@ export default function AssistantNew({
             title="Welcome to Assistant"
             icon={ChatBubbleBottomCenterTextIcon}
           />
-          <AssistantHelper>
-            <div className="mb-8 text-lg font-bold">
-              Get started with{" "}
-              <Logo className="inline-block w-14 pb-0.5 pl-1"></Logo>
-            </div>
-            <p className="my-4 text-sm text-element-800">
-              Lorem ispum dolor sit amet, consectetur adipiscing elit. You have
-              access to multiple assistants, each with their own set of skills.
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-            </p>
-            <p className="my-4 text-sm text-element-800">
-              Assistants you have access to:{" "}
-              <span className="font-bold italic">@gpt3.5-turbo</span>, and{" "}
-              <span className="font-bold italic">@claude-instant</span>.
-            </p>
-            {["admin", "builder"].includes(owner.role) && (
-              <div className="pt-4 text-center">
-                <Button
-                  variant={"primary"}
-                  icon={RobotIcon}
-                  label="Configure new Custom Assistants"
-                  onClick={() => {
-                    void router.push(`/w/${owner.sId}/builder/assistants`);
-                  }}
+
+          {/* GETTING STARTED */}
+          <div className="flex flex-col gap-8">
+            <div className="mt-6 flex flex-col gap-2">
+              <div className="flex flex-row gap-2">
+                <Icon visual={FlagIcon} size="md" />
+                <span className="text-lg font-bold">Getting started?</span>
+              </div>
+              <p className="text-element-700">
+                Using assistant is easy as asking a question to a friend or a
+                coworker.
+                <br />
+                Try it out:
+              </p>
+              <div>
+                <StartHelperConversationButton
+                  content="Hey @helper, how do I use the assistant?"
+                  variant="primary"
                 />
               </div>
-            )}
-          </AssistantHelper>
+            </div>
+
+            {/* FEATURED AGENTS */}
+            <div className="flex flex-col gap-2">
+              <div className="flex flex-row gap-2">
+                <Icon visual={HandRaisedIcon} size="md" />
+                <span className="text-lg font-bold">
+                  Meet your smart friends
+                </span>
+              </div>
+              <p className="text-element-700">
+                Dust is not just a single assistant, it’s a full team at your
+                service.
+                <br />
+                Each member has a set of specific set skills.
+              </p>
+              <p className="text-element-700">
+                Meet some of your Assistants team:
```

Comment:
Can we try adding a button see all here?

Reply:
Done!